File: LimDaeYoung/be-onboarding-project/survey-api/src/main/java/com/survey/config/WebConfig.java
package com.survey.config;

import com.survey.interceptor.HandlerInterceptor;
import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;
import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;

@Configuration
@ComponentScan("com.survey")
public class WebConfig implements WebMvcConfigurer {
    @Override
    public void addInterceptors(InterceptorRegistry registry) {
        registry.addInterceptor(new HandlerInterceptor())
                .excludePathPatterns("/");
    }
}

File: survey-api/src/main/kotlin/org/icd/surveyapi/support/config/WebConfig.kt
package org.icd.surveyapi.support.config

import org.icd.surveyapi.support.interceptor.GlobalLogInterceptor
import org.springframework.context.annotation.Configuration
import org.springframework.web.servlet.config.annotation.InterceptorRegistry
import org.springframework.web.servlet.config.annotation.WebMvcConfigurer

@Configuration
class WebConfig: WebMvcConfigurer {
    override fun addInterceptors(registry: InterceptorRegistry) {
        registry.addInterceptor(GlobalLogInterceptor())
            .addPathPatterns("/**")
    }
}